### PR TITLE
Add daemonset status check support

### PR DESCRIPTION
**Pull request by PatrickXYS**
This PR will pass tests until https://github.com/kubeflow/testing/pull/803 merged

### CODE REVIEW

File: py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py [hunk 1]
```diff
@@ -259,6 +262,17 @@ def test_kf_is_ready(record_xml_attribute, namespace, use_basic_auth,
             util.run(["kubectl", "-n", ss_namespace, "describe", "statefulsets", name])
             raise
 
+    all_daemon_sets = [(namespace, name) for name in daemon_set_names]
```
Comment (Jeffwan) [inline py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py:265]: Remind me does kubeflow manifest install any daemonset?

Comment (PatrickXYS) [reply py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py:265]: Yes for aws we have nvidia plugin, https://github.com/kubeflow/manifests/blob/master/aws/nvidia-device-plugin/base/daemonset.yaml

**Comment by PatrickXYS** [comment]
/unhold

Unhold in favor of https://github.com/kubeflow/testing/pull/803 PR merged

**Pull request merged**